### DISCUSSION TITLE
when i use databases.Database to connect mysql, how to change tuple result to dict ?

### DISCUSSION THREAD

**Discussion by weakeng** [encode/databases#306] [Q&A]
``
from aiomysql import cursors
db = databases.Database(mysql_url, min_size=1, max_size=20,cursorclass=cursors.DictCursor)
if will raise a TypeError: row must be a sequence:
          return RowProxy(metadata, row, metadata._processors, metadata._keymap)

**Comment by mecampbellsoup** [comment]
@weakeng what makes you think `cursorclass` is a supported parameter?
